contact/contact.css

```diff
@@ -0,0 +1,80 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+body{
+    font-family: 'Montserrat', sans-serif;
+}
+.header{
+    height: 94px;
+    width: 100%;
+    box-shadow: 0px 5px 16px 0px  #c2bfbf;
+    display: flex;
+}
+.header .back{
+    height: 22px;
+    margin-left: 15px;
+    margin-top: 36px;
+    font-size: 20px;
+
+}
+header .covid-19 {
+    padding-top: 25px;
+    padding-left: 20px;
+
+}
+.covid-img{
+    height: 26px;
+}
+header .dark-mode{
+    height: 25px;
+    float: right;
+    padding-right: 20px;
+    margin-left: 130px;
+    padding-top: 33px;
+
+}
+.container{
+    text-align: center;
+}
+
+.container .part{
+    width: 350px;
+    height: 510px;
+    margin-top: 47px;
+    margin-bottom: 57px;
+    text-align: center;
+    margin-left: 10px;
```

Comment:
you can write one line here instead of three lines.
like that `margin: 47px 0 57px 10px`